perfact/zodbsync/subcommand.py

```diff
@@ -150,12 +151,80 @@ def gitexec(func):
         - play back changed objects (diff between old and new HEAD)
         - unstash
         """
+        def _playback_paths(self, paths):
```

Comment:
Why do we need this embedded function? Could we pull this out of `def gitexec`?

Reply:
Shouldn't be a problem, I think this was due to the iterative development. Basically started in the `wrapper` but soon moved it out but kept it contained inside of gitexec